jangjuseong/routes/postRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+
+const postController = require('../controllers/postController');
+
+router.post('', postController.createPost);
+router.get('', postController.getPost);
```

Comment:
기본 엔드포인트도 공백으로 두지 말고 `'/'`로 표기해주세요!

```suggestion
router.post('/', postController.createPost);
router.get('/', postController.getPost);
```